src/test/scala/com/sksamuel/scapegoat/inspections/collections/MapGetAndGetOrElseTest.scala

```diff
@@ -1,10 +1,11 @@
 package com.sksamuel.scapegoat.inspections.collections
 
 import com.sksamuel.scapegoat.PluginRunner
+import org.scalatest.OneInstancePerTest
 import org.scalatest.freespec.AnyFreeSpec
 import org.scalatest.matchers.should.Matchers
 
-class MapGetAndGetOrElseTest extends AnyFreeSpec with Matchers with PluginRunner {
+class MapGetAndGetOrElseTest extends AnyFreeSpec with Matchers with PluginRunner with OneInstancePerTest {
```

Comment:
Does it mean we need `with OneInstancePerTest` everywhere now?

Reply:
I think all the other ones are run in their own instances due to the shared state from PluginRunner, so this brings this in line with all the other inspection tests.

Reply:
Exactly, only I don't understand why this did not fail before... ?!

Reply:
yeah I'm also not sure

Reply:
It's not the only one which misses that trait, let me unify that in a separate PR